README.md

```diff
@@ -116,6 +116,46 @@ This configures the `Determinator.instance` with:
 
 You may also want to configure a `determinator` helper method inside your web request scope, see below for more information.
 
+### Using over http
+
+Using the HttpRetriever will cause a request to be sent to actor tracking every time a feature is checked. The impact of this can be mitigated somewhat by having a short lived memory cache, but we're limited in
+the length of time we can cache for without some way of notifiying the cache that an item has changed.
+
+```ruby
+faraday_connection = Faraday.new("http://actor-tracking.dev") do |conn|
+  conn.basic_auth('my-service-name', 'actor-tracking-token')
+  conn.adapter Faraday.default_adapter
+end
+
+Determinator.configure(
+  retrieval: Determinator::Retrieve::HttpRetriever.new(
+    connection: faraday_connection,
+  ),
+  feature_cache: Determinator::Cache::FetchWrapper.new(
+    ActiveSupport::Cache::MemoryStore.new(expires_in: 1.minute),
+    ActiveSupport::Cache::RedisCacheStore.new
```

Comment:
Your example doesn't give an indication of how long things might exist in Redis — the total TTL is the sum of all the levels' TTLs, so it'd be good to call that out explicitly in here I think :)

Reply:
There's no need to have a TTL in redis - the explicit expire skirts around that.